Core/scripts/dirac-externals-requirements.py

```diff
@@ -69,14 +73,11 @@ def setInstallType( val ):
 
 gLogger.notice( "Requesting installation of %s" % ", ".join( [ "%s%s" % ( reqName, reqDict[ reqName ][0] ) for reqName in reqDict ] ) )
 
-from pip.index import PackageFinder
-from pip.req import InstallRequirement, RequirementSet
-from pip.locations import build_prefix, src_prefix
-
 requirement_set = RequirementSet(
-    build_dir = build_prefix,
+    build_dir = user_dir,
```

Comment:
But this is not what you want I suppose. You rather want all the externals at the same place, and available for all users no ?